pkg/health/types.go

```diff
@@ -26,16 +26,39 @@ import (
 	"time"
 )
 
+// StorageIndicatorName is the name of storage indicator
+const StorageIndicatorName = "storage"
+
+// PlatformIndicatorSuffix is the suffix for a platform type indicator
+const PlatformIndicatorSuffix = "_platforms"
+
+// indicatorNames is a list of names of indicators which will be registered with default settings
+// as part of default health settings, this will allow binding them as part of environment.
+// If an indicator is registered but not specified in this list, it will be configured with
+// default settings again, but this defaults could be overridden only via application.yml,
+// env variables and pflags won't have any effect. If an indicator is specified in this list
+// but later not registered nothing will happen.
+var indicatorNames = [...]string{
+	StorageIndicatorName,
+	"kubernetes" + PlatformIndicatorSuffix,
```

Comment:
SM must not care about different platform types in the code. These should be a configuration

Reply:
If we don't mention the indicator names here somwhow then they cannot be configured via the configuration